prometheus-exporters-formula/prometheus-exporters-formula.changes

```diff
@@ -1,3 +1,5 @@
+- Add support for older schema without proxy configuration
+
```

Comment:
Here we will likely need a new date header (like the one below), before submitting it to OBS?

Reply:
Done